cmd/manager/main.go

```diff
@@ -4,140 +4,152 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"github.com/operator-framework/operator-marketplace/pkg/builders"
-	v1 "k8s.io/api/apps/v1"
-	v12 "k8s.io/api/core/v1"
-	utilerrors "k8s.io/apimachinery/pkg/util/errors"
-	"net/http"
 	"os"
 	"runtime"
-	"strings"
-	"time"
 
-	"github.com/operator-framework/operator-marketplace/pkg/metrics"
-	"github.com/operator-framework/operator-marketplace/pkg/migrator"
+	"github.com/sirupsen/logrus"
 
 	apiconfigv1 "github.com/openshift/api/config/v1"
-	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/signals"
```

Comment:
Ended up vendoring the signal handling package from OLM into pkg/signals and removing the OLM dependency entirely. IIRC, there's a controller runtime signal handling library too that we may want to explore.

Reply:
Nice!

> A little copying is better than a little dependency.

https://go-proverbs.github.io/